javaagent-spi/src/main/java/io/opentelemetry/javaagent/spi/exporter/MetricServer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.spi.exporter;
+
+import io.opentelemetry.sdk.metrics.export.MetricProducer;
+import java.util.Properties;
+
+/**
+ * A {@link MetricServer} acts as the bootstrap for metric exporters that use {@link MetricProducer}
+ * to consume the metrics.
+ *
+ * <p>Implementation of {@link MetricServer} must be registered through the Java SPI framework.
+ */
+public interface MetricServer {
+
+  /**
+   * Initialize the metric server that pull metric from the {@link MetricProducer}.
+   *
+   * @param producer The metric producer
+   * @param config The configuration
+   */
+  void configure(MetricProducer producer, Properties config);
```

Comment:
Do you think `confugure` is a better name than `start` as Trask proposed?

Reply:
my bad I didn't pay attention. Either one if fine with me, I will rename it to start.